shell-pipelines/sort-uniq-head-tail/script-02.sh

```diff
@@ -5,3 +5,11 @@ set -euo pipefail
 # The input for this script is the scores-table.txt file.
 # TODO: Write a command to output scores-table.txt, with lines sorted by the person's first score, descending.
 # The first line of your output should be "Basia London 22 9 6" (with no quotes).
+sort -nrk3 scores-table.txt
```

Comment:
This works, but personally I tend to separate out flags which take values to make it more clear which values are associated with which flag, i.e. I would probably write `sort -nr -k3 scores-table.txt` or `sort -n -r -k3 scores-table.txt` just to make it a little more clear the 3 is attached to the `-k` :)